examples/dropwizard/README.md

```diff
@@ -0,0 +1,50 @@
+# Containerize a [Dropwizard](https://dropwizard.io) application with Jib
+
+Starter Maven template generated with [`dropwizard-archetypes`](https://github.com/dropwizard/dropwizard/tree/master/dropwizard-archetypes)
```

Comment:
Is this first section explaining how the example was generated rather than what the user should do to follow the guide? If so, I would recommend placing a header for that like "## How this example was created" or something similar.